src/main/java/uk/gov/pay/products/resources/GatewayAccountResource.java

```diff
@@ -1,54 +0,0 @@
-package uk.gov.pay.products.resources;
-
-import com.fasterxml.jackson.databind.JsonNode;
-import com.google.inject.Inject;
-import io.dropwizard.jersey.PATCH;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import uk.gov.pay.products.model.PatchRequest;
-import uk.gov.pay.products.service.GatewayAccountFactory;
-import uk.gov.pay.products.validations.GatewayAccountRequestValidator;
-
-import javax.ws.rs.Consumes;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
-import javax.ws.rs.core.Response;
-
-import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
-import static javax.ws.rs.core.Response.Status.ACCEPTED;
-import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
-
-@Path("/v1/api")
-public class GatewayAccountResource {
-
-    private static final Logger logger = LoggerFactory.getLogger(GatewayAccountResource.class);
-
-    private final GatewayAccountRequestValidator requestValidator;
-    private final GatewayAccountFactory gatewayAccountFactory;
-
-    @Inject
-    public GatewayAccountResource(GatewayAccountRequestValidator requestValidator, GatewayAccountFactory gatewayAccountFactory) {
-        this.requestValidator = requestValidator;
-        this.gatewayAccountFactory = gatewayAccountFactory;
-    }
-
-    @PATCH
-    @Path("/gateway-account/{gatewayAccountId}")
```

Comment:
Just to confirm: we do not need this endpoint anymore because it was used for service name updates only and we do not want to have gateway account update functionality anymore?

Reply:
Yes, that's correct. We stored service name in the original design as `products` was supposed to be running in PaaS environment. Then we moved `products` to our AWS environment but never updated the code. Keeping service name in sync is a bit messy. When we decided that we're going to call `adminusers` for the service object so we can implement custom branding, we also took the opportunity to use the same service object as the source of truth. Hence there is no need to keep updating the service name (which was already out of sync, anyway).